.github/workflows/build.yaml

```diff
@@ -0,0 +1,36 @@
+# This is a starting workflow for building with GitHub Actions
+name: Build
+
+on:
+  push:
+  pull_request:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      # Check out code
+      - uses: actions/checkout@v4
+      - uses: actions/setup-go@v5
+      # Set up the build environment
+      - name: Copy our kickstart file over so it gets picked up
+        run: mkdir faDockerComposeFilePath/ && cp -r .github/kickstart faDockerComposeFilePath/ && cp .github/kickstart/kickstart.json faDockerComposeFilePath/kickstart/k2.json && cp .github/kickstart/kickstart.json .github/kickstart/k2.json
+      - name: Start FusionAuth
+        uses: fusionauth/fusionauth-github-action@v1
+        with:
+          FUSIONAUTH_APP_KICKSTART_FILENAME: k2.json
+          FUSIONAUTH_VERSION: 1.51.1
```

Comment:
Do we do something like this for similar GH actions? If so, I wonder if we should be using latest? 
The latest can be obtained by using `https://account.fusionauth.io/api/latest-version` if that is helpful.

Reply:
You mean the kickstart? We have just started with that. 

If you mean the version, looks like if I don't set it [the default is latest](https://github.com/marketplace/actions/fusionauth-action) so I should remove the specific version.